utilities/test_suite/README.md

```diff
@@ -59,37 +28,45 @@ This repository contains four test suites for the AMD ROCm Performance Primitive
   make -j$nproc
```

Comment:
update the cmake flags to reflect this: https://github.com/ROCm/rocAL/blob/2f9a7fd4ef3b89acaf639592bce494907eb7b24a/rocAL-setup.py#L362
the current one doesn't work with the test suite